src/main/java/jp/co/soramitsu/sora/crypto/ProofProxy.java

```diff
@@ -0,0 +1,17 @@
+package jp.co.soramitsu.sora.crypto;
+
+import java.time.Instant;
+
+public interface ProofProxy {
+
+  Instant getCreated();
+
+  void setCreated(Instant dateTime);
```

Comment:
This is not critical

Setter methods in interface tell that the implementation is automatically mutable therefore not necessarily thread-safe
Instead, it is a good idea to have subtype of `ProofProxy` which will declare mutating methods, like:

```java
public interface ProofProxyMutable extends ProofProxy {
  void setCreated(Instant dateTime);
  void setSignatureValue(byte[] o);
}
```

Reply:
won'tfix.

It is good idea, but this concern will be valid when we need to care about concurrent environment.